### PR TITLE
OpenTelemetry metrics (and the rest of grafeas/voucher)

### PR DESCRIPTION
**Pull request by thepwagner**
Synchronize `Shopify/voucher` to `grafeas/voucher:main`.

The driver for this change is getting https://github.com/grafeas/voucher/pull/68 into the fork that Shopify uses.
I opted to implement that feature and merge `main`, because once we action https://github.com/grafeas/voucher/issues/47 , there is no longer a reason for Shopify to maintain a fork 🎉 .

You should see two types of changes in this diff:
* Adding metrics
* Inoffensive logging/release nits that we never backported

### CODE REVIEW

File: v2/cmd/voucher_server/README.md [hunk 1]
```diff
@@ -57,13 +57,19 @@ Below are the configuration options for Voucher Server:
 | `ejson`              | `dir`                        | The path to the ejson keys directory.                                                                 |
 | `ejson`              | `secrets`                    | The path to the ejson secrets.                                                                        |
 | `sops`               | `file`                       | The path to the SOPS secrets.                                                                         |
+<<<<<<< HEAD
```
Comment (lynnsh) [inline v2/cmd/voucher_server/README.md]: Is this left from git merge?

File: v2/cmd/voucher_server/TUTORIAL.md [hunk 2]
```diff
@@ -14,8 +14,12 @@ Make sure that you update the `ejson` or `sops` specific blocks to point to the
 
 ## Create secrets
 
+<<<<<<< HEAD
```
Comment (lynnsh) [inline v2/cmd/voucher_server/TUTORIAL.md]: git merge leftover here as well

File: v2/suite.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ func runner(ctx context.Context, name string, check Check, imageData ImageData,
 //
 // For example, if a Suite has the "diy" and "nobody" tests, calling
 //
-// Run(imageData)
+//	Run(imageData)
```
Comment (lynnsh) [inline v2/suite.go:61]: nit: not sure why this change is needed?

**Review by lynnsh — COMMENTED** [review]
LGTM, there is just some git merge leftovers left in README.md and TUTORIAL.md.

File: v2/suite.go (hunk 3 above)
Comment (thepwagner) [reply v2/suite.go:61]: Originally, it was 4 spaces.

* In https://github.com/grafeas/voucher/commit/af83251a3ef0b7f5dadf7c47a2bb3d3277c1b8e6 , we ran `gofmt` and in the `grafeas/voucher` fork it was replaced with a tab.
* In https://github.com/Shopify/voucher/commit/e5ab806c69b064bd3b1331bd54f4dbbb51134f40 , we ran `gofmt` on the `shopify/voucher` fork and it was replaced with a single space.

The actual whitespace is definitely moot, but it serves the goal of aligning these repositories and making it easier to migrate codez between them.

File: v2/cmd/voucher_server/TUTORIAL.md [hunk 4]
```diff
@@ -14,12 +14,7 @@ Make sure that you update the `ejson` or `sops` specific blocks to point to the
 
 ## Create secrets
 
-<<<<<<< HEAD
-(Deprecated, clair is no longer supported)
```
Comment (lynnsh) [inline v2/cmd/voucher_server/TUTORIAL.md:17]: Would we want to mention somewhere that `clair` is no longer supported?

**Review by lynnsh — APPROVED** [review]

File: v2/cmd/voucher_server/TUTORIAL.md (hunk 4 above)
Comment (thepwagner) [reply v2/cmd/voucher_server/TUTORIAL.md:17]: We were OK with removing the mention in https://github.com/grafeas/voucher/pull/66 ; so I don't think it's a big issue.

I think it's easy enough to see this being removed - https://github.com/search?q=repo%3Agrafeas%2Fvoucher%20clair&type=code , I don't feel bad not leaving a not that a feature that nobody used is no longer supported 😆 .
